### PR TITLE
pr_test

### PR DESCRIPTION
**Pull request by kidoo312**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/kidoo312/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: pr_test [hunk 1]
```diff
@@ -0,0 +1 @@
+pr_test
```
Comment (kidoo312) [inline pr_test]: line note test !fix

Comment (kidoo312) [inline pr_test]: !fixed

File: pr_test2 [hunk 2]
```diff
@@ -0,0 +1,24 @@
+test
+test
+test
+test
+test
+test
+test
+test
+test
```
Comment (kidoo312) [inline pr_test2]: !resolve 
line 9 ~ line 15 까지 수정

**Pull request merged**